app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -98,9 +98,8 @@ public Result resendEmailVerification() throws Exception {
     }
 
     public Result requestResetPassword() throws Exception {
-        JsonNode json = requestToJSON(request());
         Email email = parseJson(request(), Email.class);
-        Study study = getStudyOrThrowException(json);
+        Study study = studyService.getStudy(email.getStudyIdentifier());
```

Comment:
So, there weren't tests for other methods in AuthenticationController either, so I've added them as well, and they all check the kill switch now if the User-Agent string is present.